README.md

```diff
@@ -219,7 +219,7 @@ Please take a look at the [Contribution Guidelines](https://github.com/meilisear
 
 ### Community Blog Posts
 * [Meilisearch: A Minimalist Full-Text Search Engine](https://tech.marksblogg.com/meilisearch-full-text-search.html)
-* :cn: [Vuepress全文搜索终极版-基于开源搜索引擎meilisearch的全接入指南](https://wiki.eryajf.net/pages/dfc792/) - Vuepress Full Text Search Ultimate Edition-Full Access Guide Based on Open Source Search Engine meilisearch
+* :cn: [Vuepress全文搜索终极版-基于开源搜索引擎meilisearch的接入配置指南](https://wiki.eryajf.net/pages/dfc792/) - A guide on Meilisearch-based full-text search in Vuepress
```

Comment:
Thanks for the changes. I'm sorry for being annoying, but I just thought that as a guide it should probably belong to the `Community guides` section? If you agree, I'll make the changes, you've already done a lot :)